view/frontend/templates/init.phtml

```diff
@@ -4,6 +4,8 @@ $productListMode = $block->getProductListMode();
 $enable = $block->isEnable(); 
 $delay = $helper->getConfig('lazyloadinginfinitescroll/instances/delay');
 $delay = $delay?(int)$delay:600;
```

Comment:
more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.